Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -91,12 +91,12 @@ def tests
       end
 
       args = ["-I", HOMEBREW_LIBRARY_PATH/"test"]
-      args += %w[
+      args += %W[
         --color
         --require spec_helper
         --format progress
         --format ParallelTests::RSpec::RuntimeLogger
-        --out tmp/parallel_runtime_rspec.log
+        --out #{ENV["HOME"]}/.cache/parallel_rspec/runtime.log
```

Comment:
@reitermarkus It does but I think it's misleading; the other files are actually temporary whereas this runtime log is (like RuboCop's) a cache entry that massive speeds things up if it exists.